src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package controller;
+
+import Entity.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import service.UserService;
+
+@RestController
+public class UserController {
```

Comment:
If you're intending to create REST-ful interface, it is good to write API routes based on verbs.
Ex.  
1. POST `/user` -> Create user
2. GET `/user` -> Get all users
3. GET `/user/{email}` -> Get a user 


This way, frontend has less work keeping the API route names consistent (The API route name is the same for anything wrt User)